ddtrace/constants.py

```diff
@@ -1,23 +1,8 @@
-FILTERS_KEY = "FILTERS"
-SAMPLE_RATE_METRIC_KEY = "_sample_rate"
 SAMPLING_PRIORITY_KEY = "_sampling_priority_v1"
 ANALYTICS_SAMPLE_RATE_KEY = "_dd1.sr.eausr"
-SAMPLING_AGENT_DECISION = "_dd.agent_psr"
-SAMPLING_RULE_DECISION = "_dd.rule_psr"
-SAMPLING_LIMIT_DECISION = "_dd.limit_psr"
 ORIGIN_KEY = "_dd.origin"
-HOSTNAME_KEY = "_dd.hostname"
-ENV_KEY = "env"
 VERSION_KEY = "version"
 SERVICE_KEY = "service.name"
 SERVICE_VERSION_KEY = "service.version"
 SPAN_KIND = "span.kind"
 SPAN_MEASURED_KEY = "_dd.measured"
-KEEP_SPANS_RATE_KEY = "_dd.tracer_kr"
-
-NUMERIC_TAGS = (ANALYTICS_SAMPLE_RATE_KEY,)
-
-MANUAL_DROP_KEY = "manual.drop"
-MANUAL_KEEP_KEY = "manual.keep"
-
-LOG_SPAN_KEY = "__datadog_log_span"
```

Comment:
From sync: we should have a deprecation warning here for the ones that are going away.

Cons are that it might be noisy for people who are using the proper public constants